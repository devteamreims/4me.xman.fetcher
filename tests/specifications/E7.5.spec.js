import request from 'supertest';

import app from '../../index';

test('have a /status route', () => {
  return request(app)
    .get('/status')
    .expect(200)
    .expect(res => {
      expect(res.body).toMatchSnapshot();
    });
});
