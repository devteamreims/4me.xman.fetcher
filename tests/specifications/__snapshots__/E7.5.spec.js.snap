exports[`test have a /status route 1`] = `
Object {
  "lastRequest": Object {
    "error": null,
    "rawResponse": null,
    "when": null
  },
  "version": "0.1.2"
}
`;
